ArduCopter/mode_althold.cpp

```diff
@@ -21,6 +21,13 @@ bool Copter::ModeAltHold::init(bool ignore_checks)
 // should be called at 100hz or more
 void Copter::ModeAltHold::run()
 {
+    enum AltHoldModeState {
```

Comment:
can we leave this change out?  I've got a big PR coming that totally changes the althold states